tests/contrib/funders/test_funders_resource.py

```diff
@@ -0,0 +1,179 @@
+
+    res = client.get(f"{prefix}/{id_}", headers=h)
+    assert res.status_code == 200
+    assert res.json["id"] == id_
+    # Test links
+    assert res.json["links"] == {
+        "self": "https://127.0.0.1:5000/api/funders/fund"
+    }
+
+
+def test_funders_search(client, example_funder, h, prefix):
+    """Test a successful search."""
+    res = client.get(prefix, headers=h)
+
+    assert res.status_code == 200
+    assert res.json["hits"]["total"] == 1
+    assert res.json["sortBy"] == "name"
+
+
+def _create_funders(service, identity):
+    """Create dummy funders with similar names/titles."""
+    funders = [
+        {
+            "id": "cern",
+            "name": "CERN",
+            "country": "CH",
+            "title": {
+                "en": "European Organization for Nuclear Research",
+                "fr": "Conseil Européen pour la Recherche Nucléaire"
+            }
+        },
+        {
+            "id": "other",
+            "name": "OTHER",
+            "country": "CH",
+            "title": {
+                "en": "CERN"
+            }
+        },
+        {
+            "id": "cert",
+            "name": "CERT",
+            "country": "CH",
+            "title": {
+                "en": "Computer Emergency Response Team",
+                "fr": "Équipe d'Intervention d'Urgence Informatique"
+            }
+        },
+        {
+            "id": "nu",
+            "name": "Northwestern University",
+            "country": "US",
+            "title": {
+                "en": "Northwestern University",
+            }
+        }
+    ]
+    for fun in funders:
+        service.create(identity, fun)
+
+    Funder.index.refresh()  # Refresh the index
+
+
+def test_funders_suggest_sort(client, h, identity, prefix, service):
+    """Test a successful search."""
+    _create_funders(service, identity)
+
+    # Should show 3 results, but id=cern as first due to name
+    res = client.get(f"{prefix}?suggest=CER", headers=h)
+    assert res.status_code == 200
+    assert res.json["hits"]["total"] == 3
+    assert res.json["hits"]["hits"][0]["id"] == "cern"
+    assert res.json["hits"]["hits"][1]["id"] == "cert"
+    assert res.json["hits"]["hits"][2]["id"] == "other"
+
+    # Should show 0 results since scheme is not searchable
+    res = client.get(f"{prefix}?suggest=nucléaire", headers=h)
+    assert res.status_code == 200
+    assert res.json["hits"]["total"] == 1
+
+
+def test_funders_delete(client_with_credentials, example_funder, h, prefix):
+    """Test a successful delete."""
+    id_ = example_funder.id
+    res = client_with_credentials.delete(f"{prefix}/{id_}", headers=h)
+    assert res.status_code == 204
+
+    res = client_with_credentials.get(f"{prefix}/{id_}", headers=h)
+    assert res.status_code == 410
+
+
+def test_funders_update(
+    client_with_credentials, example_funder, funder_full_data, h, prefix
+):
+    """Test a successful update."""
+    id_ = example_funder.id
+    new_name_ = "updated"
```

Comment:
```suggestion
    new_name = "updated"
```

We put `_` after `id` because it is a reserved python keyword, for other variables is not needed. Ref https://docs.python.org/3/library/functions.html#id

Reply:
thanks for the tip!